packages/generate-cpf/src/index.js

```diff
@@ -0,0 +1,22 @@
+import { CPF_LENGTH, STATES_CODE } from './constants';
+
+const createCheckSum = (cpfStart, base) =>
```

Comment:
instead of pass this base number we can pass an array. I was reading how cnpj validation works and the multiplication use diferent weights for each digit

![image](https://user-images.githubusercontent.com/12724212/42061682-5cd85686-7b01-11e8-9fc8-a2ff79ab8338.png)

what you guys think? and if the argument is only a number we can use only him as the weight

Reply:
Done! [faf9ebd](https://github.com/hyanmandian/brazilian-utils/pull/18/commits/faf9ebd8bfc71192fa2cda97158219c5a868223a)